README.md

```diff
@@ -91,47 +88,13 @@ pip install -e ./instrumentation/opentelemetry-instrumentation-{instrumentation}
 For additional exporter and instrumentation packages, see the 
 [`opentelemetry-python-contrib`](https://github.com/open-telemetry/opentelemetry-python-contrib) repository.
 
-## Running Performance Tests
-
-This section provides details on how to reproduce performance tests results on your own
-machine.
-
-### Resource Usage Tests
-
-1. Install scalene using the following command
-
-```sh
-pip install scalene
-```
-
-2. Run the `scalene` tests on any of the example Python programs
-
-```sh
-scalene opentelemetry-<PACKAGE>/tests/performance/resource-usage/<PATH_TO_TEST>/profile_resource_usage_<NAME_OF_TEST>.py
-```
-
-## Python Version Support Addition and Removal
-
-This project supports the latest Python versions. As new Python versions are released, support for them is added and
-as old Python versions reach their end of life, support for them is removed.
-
-We add support for new Python versions no later than 3 months after they become stable.
-
-We remove support for old Python versions 6 months after they reach their [end of life](https://devguide.python.org/devcycle/#end-of-life-branches).
-
-## Documentation
-
-The online documentation is available at https://opentelemetry-python.readthedocs.io/.
-To access the latest version of the documentation, see
-https://opentelemetry-python.readthedocs.io/en/latest/.
-
 ## Contributing
 
 For information about contributing to OpenTelemetry Python, see [CONTRIBUTING.md](CONTRIBUTING.md).
 
 We meet weekly on Thursdays at 9AM PST. The meeting is subject to change depending on contributors' availability. Check the [OpenTelemetry community calendar](https://calendar.google.com/calendar/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com) for specific dates and Zoom meeting links.
 
-Meeting notes are available as a public [Google doc](https://docs.google.com/document/d/1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs/edit). For edit access, get in touch on [GitHub Discussions](https://github.com/open-telemetry/opentelemetry-python/discussions).
+Meeting notes are available as a public [Google doc](https://docs.google.com/document/d/1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs/edit).
 
 Approvers ([@open-telemetry/python-approvers](https://github.com/orgs/open-telemetry/teams/python-approvers)):
```

Comment:
Should we add @sanketmehta28  and @ashu658 to the list of approvers? Also wondering if we should remove @oxeye-nikolay since not much activity from him regarding this repo?

Reply:
They are already part of the list. I would encourage people to remove themselves if they don't see themselves contributing to the project both @ashu658 and @oxeye-nikolay.